runs/comparisons/suprb_all_tuning.py

```diff
@@ -47,7 +47,7 @@ def run(problem: str, job_id: str):
 
     estimator = SupRB(
         rule_generation=rs.RandomSearch(),
-        solution_composition=ga.GeneticAlgorithm(),
+        solution_composition=abc.ArtificialBeeColonyAlgorithm(),
```

Comment:
why change the default?

Reply:
Irrelevant as it gets overridden by the tuning. Was changed for testing purposes